### DISCUSSION TITLE
Flynt Slider Image

### DISCUSSION THREAD

**Discussion by purchasingtoolpak** [flyntwp/flynt#357] [Q&A] [answered]
Noticed Flynt uses Swiper JS for the Slider Image Component.

There are lots of options with Swiper JS - for example I would like 3 slides per page. 

What would be the best way to access Slider JS options to customize?

**Comment by koukouchris** [comment, marked as answer]
Hi @purchasingtoolpak for SwiperJS options in SliderImages component, you can edit the config and add your own additional swiper options (like `slidesPerView`) in the [script.js](https://github.com/flyntwp/flynt/blob/master/Components/SliderImages/script.js#L41) file, where swiper is initialised.

**Reply by purchasingtoolpak** [reply]
OK, that worked well.

THANKS
